documentation/src/docs/asciidoc/user-guide/appendix.adoc

```diff
@@ -50,10 +50,17 @@ artifacts are deployed to Sonatype's {snapshot-repo}[snapshots repository] under
   `junit-platform-runner`::
     Runner for executing tests and test suites on the JUnit Platform in a JUnit 4
     environment. See <<running-tests-junit-platform-runner>> for details.
+  `junit-platform-suite`::
+    JUnit Platform Suite artifact that transitively pulls in dependencies on
+    `junit-platform-suite-api`, `junit-platfomr-suite-engine` for simplified
+    dependency management in build tools such as Gradle and Maven.
   `junit-platform-suite-api`::
     Annotations for configuring test suites on the JUnit Platform. Supported by the
-    <<running-tests-junit-platform-runner,JUnitPlatform runner>> and possibly by
-    third-party `TestEngine` implementations.
+    <<junit-platform-suite-engine,JUnit Platform Suite Engine>> and
+    <<running-tests-junit-platform-runner,JUnitPlatform runner>>.
```

Comment:
I've intentionally remove the reference to other test engines. Looking at the current implementations I don't think it is feasible for other test engines to replicate this functionality without using JUnits internals.